site/docs/components/toggle-button/examples.mdx

```diff
@@ -10,62 +10,136 @@ data:
+
+## Best practices
+
+- **Visual hierarchy**: Changing the appearance to bordered can reduce the prominence of the toggle button when in a selected state.
+
+<LivePreview componentName="toggle-button" exampleName="Appearance" />
+
+## Disabled
+
+You can disable toggle buttons if required by context or user permissions using the `disabled` prop. Disabling a toggle button will prevent the user from interacting with it and suppress all functionality.
+
+<LivePreview componentName="toggle-button" exampleName="Disabled" />
+
 ## Toggle button group
 
-`ToggleButtonGroup` allows users to make a mutually exclusive selection from a set of related commands—with only one option selected at a time.
+A `ToggleButtonGroup` allows users to make a mutually exclusive selection from a set of related commands—with only one option selected at a time.
 
 ### Best practices
 
-- A toggle button group should always contain two or more options related by function or context. Examples include text alignment (left, center, right) or speed selection (slow, medium, fast).
-- Use text description in addition to or instead of an icon if the options in the toggle button group are high priority. Users will be less likely to overlook the options.
+- **Number of options**: A toggle button group should always contain two or more options related by function or context. Examples include text alignment (left, center, right, justified) or risk level (high, medium, low).
+- **Text and icon**: Use text description in addition to or instead of an icon if the options in the toggle button group are high priority. Users will be less likely to overlook the options.
 
 <LivePreview
   componentName="toggle-button"
   exampleName="ToggleButtonGroupDefault"
 />
 
-## Icon only
+## Group orientation
+
+By default, a toggle button group has horizontal alignment. To present the options in a stacked layout, orientate the group vertically using the `orientation=”vertical”` prop.
+
+When oriented vertically, the group will resize to fit the longest toggle button, with all other buttons in the group extending to fill the same width.
+
+<LivePreview
+  componentName="toggle-button"
+  exampleName="ToggleButtonGroupVertical"
+/>
+
+## Read-only group
+
+Setting the toggle button group as read-only using the `readOnly` prop will render the toggle button group in a non-interactive state and with read-only styles. This is useful for scenarios where users need to view the current selection, but may not have the right permissions to change it.
+
+If a toggle button is in a selected state within a read-only toggle button group, it will retain its visual style to indicate selection and remain focusable via keyboard navigation, ensuring that users can perceive the selection through assistive technologies.
+
+<LivePreview
+  componentName="toggle-button"
+  exampleName="ToggleButtonGroupReadOnly"
```

Comment:
There's a spelling error in the readonly example. 'selectd' should be 'selected'